internal/osutils/user/user_win.go

```diff
@@ -0,0 +1,23 @@
+//go:build windows
```

Comment:
This is not necessary. Go's `os.UserHomeDir()` accounts for Windows, MacOS, and Linux.